core/common/fd_usage.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <cstdlib>
+
+namespace fc {
+  inline size_t fdUsage();
```

Comment:
Please, add description like `Returns the number of fds opened by the current process.`